src/services/comparison/utils.ts

```diff
@@ -1,13 +1,26 @@
 import isEmpty from 'lodash/isEmpty'
+import { z } from 'zod'
 
-function _setFileLabel({ isNewFile, isRenamedFile, isDeletedFile }) {
+import { ImpactedFileSchema } from './useComparisonForCommitAndParent'
+
+function _setFileLabel({
+  isNewFile,
+  isRenamedFile,
+  isDeletedFile,
+}: {
+  isNewFile: boolean
+  isRenamedFile: boolean
+  isDeletedFile: boolean
+}): string | null {
   if (isNewFile) return 'New'
   if (isRenamedFile) return 'Renamed'
   if (isDeletedFile) return 'Deleted'
   return null
 }
 
-export function transformImpactedFileData(impactedFile) {
+export function transformImpactedFileData(
+  impactedFile: z.infer<typeof ImpactedFileSchema>
```

Comment:
instead of this, you can export the `typeof` in the hook. This still works but that's generally what everyone else has been doing. 

I think we have a `export type ImpactedFileType` already in `useCommit`

Reply:
That's a good call out, swapped in [dc7e63c](https://github.com/codecov/gazebo/pull/2857/commits/dc7e63c895fbc6af732eaea8cffd86868c0696e3)